AGENTS.md

```
@@ -0,0 +1,18 @@
+- DocBook-based manuals are in `doc/`
+
+## Package management
+- Use `uv pip install --system -e .` to install dependencies defined in `setup.py`
+- Tool configurations live in `pyproject.toml`
+
+## Testing
+- Run `python tests/runtests.py --config=tests/unittest.conf --no-interactive`
+  to execute the broker test suite
+- Some tests require additional environment modules; see `setup-dev-env.sh`
+
+## Linting and formatting
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)